buildSrc/build.gradle

```diff
@@ -46,7 +46,7 @@ task sourcesJar(type: Jar) {
     classifier = 'sources'
 }
 
-def publishVersion = '0.20.7'
+def publishVersion = '0.20.8-SNAPSHOT'
```

Comment:
Isn't it typical to bump the version to the next release snapshot? This makes it easier to test changes locally.